Kernel/Bus/USB/USBDevice.h

```diff
@@ -75,6 +77,7 @@ class Device : public AtomicRefCounted<Device> {
 
     NonnullLockRefPtr<USBController> m_controller;
     NonnullOwnPtr<ControlPipe> m_default_pipe; // Default communication pipe (endpoint0) used during enumeration
+    OwnPtr<Pipe> m_interrupt_in_pipe;
```

Comment:
Please store the pipe in the `USBMouseDevice`, instead of adding code to `USBDevice`